docs/outlook/item-data.md

```diff
@@ -51,12 +51,12 @@ For the remaining item-level properties available in compose forms, because the
 
 If your Outlook add-in is activated in read forms, you can get an Exchange callback token. This token can be used in server-side code to access the full item via Exchange Web Services (EWS).
 
-By specifying the **ReadItem** permission in the add-in manifest, you can use the [mailbox.getCallbackTokenAsync](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox#methods) method to get an Exchange callback token, the [mailbox.ewsUrl](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox#properties) property to get the URL of the EWS endpoint for the user's mailbox, and [item.itemId](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item#properties) to get the EWS ID for the selected item. You can then pass the callback token, EWS endpoint URL, and the EWS item ID to server-side code to access the [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation, to get more properties of the item.
+By specifying the **read item** permission in the add-in manifest, you can use the [mailbox.getCallbackTokenAsync](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox#methods) method to get an Exchange callback token, the [mailbox.ewsUrl](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox#properties) property to get the URL of the EWS endpoint for the user's mailbox, and [item.itemId](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item#properties) to get the EWS ID for the selected item. You can then pass the callback token, EWS endpoint URL, and the EWS item ID to server-side code to access the [GetItem](/exchange/client-developer/web-service-reference/getitem-operation) operation, to get more properties of the item.
```

Comment:
Consider linking **read item** to its relevant section in [Understanding Outlook add-in permissions](https://review.learn.microsoft.com/en-us/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions?branch=pr-en-us-3670).